test/metabase/dashboard_subscription_test.clj

```diff
@@ -220,7 +221,10 @@
 ;;; |                                                     Tests                                                      |
 ;;; +----------------------------------------------------------------------------------------------------------------+
 
-(def execute-dashboard (requiring-resolve 'metabase.notification.payload.execute/execute-dashboard))
+(defn execute-dashboard
+  [& args]
+  (let [dashboard-result (apply (requiring-resolve 'metabase.notification.payload.execute/execute-dashboard) args)]
```

Comment:
Why is this dynamic require required? I can't imagine there's a cyclic dependency to a test.